style-guide.md

```diff
@@ -34,9 +34,11 @@ import scala.collection.mutable.ArrayBuilder
   ... ArrayBuilder[Byte] ...
 ```
 
+ - Prefer `None`/`Some` to `Option.empty`/`Option`, respectively.
```

Comment:
Just a remark, `Option(a)` and `Some(a)` aren't the same.  `Option(null: T): Option[T] == None` while `Some(null: T): Option[T] == Some(null: T)`.